interpreter_book/src/lexer/lexer.go

```diff
@@ -0,0 +1,58 @@
+package lexer
+
+import (
+	"github.com/Asalle/monkey-interpreter/token"
```

Comment:
it might be helpful to keep this more general for when others try to use or add to this code e.g. `monkey-interpreter/token`

Reply:
this is true, I completely forgot that in the end we'll merge this into the part repo "wwgberlin" and it will totally break the build, silly me! :D 
I'll fix it up in next commit!